test-cases/longevity/longevity-large-collections-48h.yaml

```diff
@@ -0,0 +1,19 @@
+# 48 hours
+test_duration: 2880
+prepare_write_cmd: "cassandra-stress write cl=ALL n=1000000  -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..200200300 -log interval=15"
```

Comment:
Why do you need this stage?

Reply:
Some nemesis failed for lack of keyspace1.standard1, so I added it.

Reply:
no need, you can remove it, let them failed.